src/tests/chain-jsonrpc.test.ts

```diff
@@ -1,8 +1,9 @@
-import { JsonRpc } from './chain-jsonrpc';
-import { RpcError } from '../chain-rpcerror';
+import { RpcError } from "../chain-rpcerror";
+import { JsonRpc } from "../chain-jsonrpc";
+import * as _ from "lodash";
```

Comment:
Please import only used method from lodash instead of all methods. Check in other places.